lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
@branan do you want to make that change ^? Also I'm wondering if we'd have the same issue with the `yaml` terminus and need to apply the same logic?